.circleci/config.yml

```diff
@@ -339,6 +339,20 @@ commands:
           path: fastlane/test_output/xctest
           destination: scan-test-output
 
+  push-to-spm:
+    steps:
+      - checkout
+      - setup-git-credentials
+      - run:
+          name: Clone purchases-ios and push to purchases-ios-spm
+          command: |
+            git clone https://github.com/RevenueCat/purchases-ios.git
+            cd purchases-ios
```

Comment:
this will always be running _from_ a clone, right? why do we do this instead of adding the other repo as an alt remote to the existing repo?

Reply:
As discussed, merging now because it's working and we are biased to shipping, will return to this as a possible future improvement.